### DISCUSSION TITLE
Page Login <p-background>

### DISCUSSION THREAD

**Discussion by eairafao** [po-ui/po-angular#1136] [Tire suas dúvidas] [answered]
Bom dia!

Espero que estejam bem.

Pessoal, gostaria de sanar uma dúvida.. é possível alterar o lado do p-background na pageLogin? ou sempre vai ficar no lado direito?

Obrigado pelo projeto,

Até logo. =)

**Comment by alinelariguet** [comment, marked as answer]
Boa tarde @eairafao 
Atualmente não é possível alterar o lado do background, ele sempre aparecerá do lado direito. A sua necessidade de colocar do lado esquerdo parte de alguma definição de UX?

**Reply by eairafao** [reply]
Boa tarde, @alinelariguet

Sim, na empresa que atuo, utilizamos um "padrão" com os lados do login. Como é nosso primeiro projeto em po-ui, tivemos questionamentos dos gestores por questão do layout padrão e surgiu essa dúvida.

Obrigado pelo rápido retorno.
